### PR TITLE
explanation is an object: text, icon, url

### PR DESCRIPTION
**Pull request by lsmith77**

### CODE REVIEW

File: tests/test_general_cases/test_api_english_singular_they/output.json [hunk 1]
```diff
@@ -10,7 +10,11 @@
             ],
             "category": "unconscious_bias",
             "end": 14,
-            "explanation": "😟 Pronouns according to binary normative",
+            "explanation": {
+                "icon": "😟",
+                "text": "Pronouns according to binary normative",
+                "url": "https://www.witty.works/en/unconscious_bias#binary_pronouns"
```
Comment (Szarad) [inline tests/test_general_cases/test_api_english_singular_they/output.json:16]: this URLs will be soon added? but anyway we are using old version in the browser extension for now?

File: app/models.py [hunk 2]
```diff
@@ -318,11 +325,30 @@ def factory(
             else lang._("rules." + subcategory + "_solution", params)
         )
 
-        explanation = (
-            explanation
-            if explanation != None
-            else lang._("rules." + subcategory + "_explanation", params)
-        )
+        if explanation != None:
```
Comment (Szarad) [inline app/models.py:328]: when this case is happening? That explanation is not None?

**Review by Szarad — APPROVED** [review]
I have just two question to clarify things for myself, but works and looks good

File: tests/test_general_cases/test_api_english_singular_they/output.json (hunk 1 above)
Comment (lsmith77) [reply tests/test_general_cases/test_api_english_singular_they/output.json:16]: yeah .. the URL format might also change still

File: app/models.py (hunk 2 above)
Comment (lsmith77) [reply app/models.py:328]: this is coming via #386

**Pull request merged**